### PR TITLE
[ISSUE #1558]module `acl` compatible with 1.6

### PR DESCRIPTION
**Pull request by geektao**
## What is the purpose of the change

fix [#1558](https://github.com/apache/rocketmq/issues/1558)  module `acl` compatible with 1.6

### CODE REVIEW

**Comment by duhenglucky** [comment]
@geektao thanks for your contribution, and it seems the CI process failed:
[ERROR] /home/travis/build/apache/rocketmq/acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java:[212,52] diamond operator is not supported in -source 1.6
  (use -source 7 or higher to enable diamond operator)
[ERROR] /home/travis/build/apache/rocketmq/acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:[276,57] diamond operator is not supported in -source 1.6

**Comment by geektao** [comment]
> @geektao thanks for your contribution, and it seems the CI process failed:
> [ERROR] /home/travis/build/apache/rocketmq/acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java:[212,52] diamond operator is not supported in -source 1.6
> (use -source 7 or higher to enable diamond operator)
> [ERROR] /home/travis/build/apache/rocketmq/acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:[276,57] diamond operator is not supported in -source 1.6
> 
> could you fix it?

I try to merge develp branch ,and fix it .

File: acl/src/main/java/org/apache/rocketmq/acl/common/Permission.java [hunk 1]
```diff
@@ -59,19 +59,19 @@ public static byte parsePermFromString(String permString) {
...
-            case "PUB|SUB":
-                return Permission.PUB | Permission.SUB;
-            case "SUB|PUB":
-                return Permission.PUB | Permission.SUB;
-            case "DENY":
-                return Permission.DENY;
-            default:
-                return Permission.DENY;
+        String permTrimString = permString.trim();
+        if ("PUB".equals(permTrimString)) {
```
Comment (zongtanghu) [inline acl/src/main/java/org/apache/rocketmq/acl/common/Permission.java]: Here,you can write the codes,such as "if (permTrimString.equals("PUB"))".

File: acl/src/main/java/org/apache/rocketmq/acl/common/Permission.java [hunk 2]
```diff
@@ -59,19 +59,19 @@ public static byte parsePermFromString(String permString) {
...
+            return Permission.PUB;
+        } else if ("SUB".equals(permTrimString)) {
+            return Permission.SUB;
+        } else if ("PUB|SUB".equals(permTrimString)) {
+            return Permission.PUB | Permission.SUB;
+        } else if ("SUB|PUB".equals(permTrimString)) {
+            return Permission.PUB | Permission.SUB;
+        } else if ("DENY".equals(permTrimString)) {
+            return Permission.DENY;
```
Comment (zongtanghu) [inline acl/src/main/java/org/apache/rocketmq/acl/common/Permission.java]: The same question as above.

**Comment by zongtanghu** [comment]
I review your pr(@geektao ) and leave some code comments.

**Pull request closed without merging**